### PR TITLE
Feat/job teaser

### PR DESCRIPTION
**Pull request by trinegl**

### CODE REVIEW

File: _includes/assets/doodle-circle.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg width="100" height="48" viewBox="0 0 100 48" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (timbru31) [inline _includes/assets/doodle-circle.svg:1]: I think it's a great idea to put the SVG through https://jakearchibald.github.io/svgomg/ :)

File: _includes/job-teaser.liquid [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="grid">
+	<section class="job-teaser col col-start-4 col-end-10 col-start-3-s col-end-11-s col-start-1-xs col-end-13-xs">
+		<h2 class="job-teaser__header">
+			We <span class="job-teaser__doodle">love</span> thinkers<br />
+			that do.
+		</h2>
+		<p class="job-teaser__txt">Ever thought of amplifying your impact to the world? Ever clicked on a call to action in a yellow circle?</p>
```
Comment (timbru31) [inline _includes/job-teaser.liquid:7]: I'd vote for `job-teaser__text` over `txt`

File: _includes/assets/doodle-circle.svg (hunk 1 above)
Comment (trinegl) [reply _includes/assets/doodle-circle.svg:1]: I think you're right :) But I'm wondering, if this should even be an SVG, it's the img tag that is massive..

File: _includes/join-us.liquid [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<section class="join">
+	<h2 class="join__header">We <span class="join__doodle">love</span> thinkers that do.</h2>
+	<p class="join__txt">Ever thought of amplifying your impact to the world? Ever clicked on a call to action in a yellow circle?</p>
+	<a class="join__link" href="#">Join us</a>
```
Comment (tam-kis) [inline _includes/join-us.liquid]: We can shape it as a circle with border-radius, add a yellow border and later animate it with css when the animation is decided, for example: https://stackoverflow.com/questions/23934749/fill-element-from-center-on-hover

Comment (trinegl) [reply _includes/join-us.liquid]: I think this is from and old commit :) Current version should already be a styled as a circle with border etc

Comment (tam-kis) [reply _includes/join-us.liquid]: Indeed :D

File: _includes/assets/doodle-circle.svg (hunk 1 above)
Comment (trinegl) [reply _includes/assets/doodle-circle.svg]: I changed to png file instead :)

File: _includes/job-teaser.liquid (hunk 2 above)
Comment (trinegl) [reply _includes/job-teaser.liquid]: Done :)

**Review by tam-kis — APPROVED** [review]

**Pull request merged**